src/libstd/time.rs

```diff
@@ -414,7 +416,7 @@ impl SystemTime {
     ///
     /// let sys_time = SystemTime::now();
     /// let difference = sys_time.duration_since(sys_time)
-    ///                          .expect("SystemTime::duration_since failed");
+    ///                          .expect("Clock may have went backwards");
```

Comment:
```suggestion
    ///                          .expect("Clock may have gone backwards");
```

though making it say "may have" might not be a good message (even for a doc example)?

Reply:
What do you propose?

I think the original text about "failed" was too general. Most likely `duration_since` is not to blame here, but it's a side-effect of the clock changing. However, I don't know if that's guaranteed to be the only cause of failure, which is why I put "may".

Is

```rust
.expect("Clock has gone backwards");
```

ok?

Reply:
I totally agree that the original message was bordering on useless (other than as a marker to where the fail happened), and looking at it again "may have" might be the better choice if there could be other chances for failure.